src/H5SM.c

```diff
@@ -288,12 +288,12 @@ H5SM__type_to_flag(unsigned type_id, unsigned *type_flag)
  *
  *-------------------------------------------------------------------------
  */
-ssize_t
-H5SM__get_index(const H5SM_master_table_t *table, unsigned type_id)
+herr_t
+H5SM__get_index(const H5SM_master_table_t *table, unsigned type_id, ssize_t *idx)
 {
-    size_t   x;
     unsigned type_flag;
-    ssize_t  ret_value = FAIL;
+    ssize_t  indx      = -1;
```

Comment:
I'd probably use found_idx instead of indx. One letter differences are easy to confuse.

Reply:
Fine w/me  :-)